compiler/plc_diagnostics/src/reporter.rs

```diff
@@ -35,6 +35,7 @@ impl ResolvedLocation {
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct ResolvedDiagnostics {
```

Comment:
ResolvedDiagnostic was a diagnostic where we know the location in file. It might have become redundant. I'll create an issue where we could tackle this.